mlflow/types/llm.py

```diff
@@ -323,11 +344,12 @@ class ChatResponse(_BaseDataclass):
     """
 
     choices: List[ChatChoice]
-    usage: TokenUsageStats
+    usage: Optional[TokenUsageStats] = None
     id: Optional[str] = None
     model: Optional[str] = None
     object: Literal["chat.completion"] = "chat.completion"
     created: int = field(default_factory=lambda: int(time.time()))
+    metadata: Optional[Dict[str, str]] = None
```

Comment:
The field doc is missing

Reply:
actually it's not ready for review yet, i'm still fixing some stuff 😆

i will add docs and fix tests first